### DISCUSSION TITLE
Two sets of villager brain ticks

### DISCUSSION THREAD

**Discussion by 1ByteBit** [PurpurMC/Purpur#1184] [__Dismissed__] [answered]
Currently we have the option to change villager brain ticks, and decide whether its for only if the server is lagging. My suggestion is to have two configs:
Villager brain ticks for when the server isn't lagging
Villager brain ticks for when the server does lag
This can allow people to optimize villagers for normal server operations, as well as allowing them to much more heavily optimize (for lack of a better word) them when the server lags

**Comment by BillyGalbreath** [comment, marked as answer]
I actually want to get rid of villager brain ticks entirely. It's a really old patch that was just a simple cheap "fix" at the time. We have much better options available to us these days.

https://docs.papermc.io/paper/reference/world-configuration#sensor

https://docs.papermc.io/paper/reference/world-configuration#behavior

**Reply by 1ByteBit** [reply]
Ah I see, thanks for the response, I will definitely be using those instead

**Reply by BillyGalbreath** [reply]
If you join our Discord you can find a list of all the sensors and their default values pinned in the help channel

**Reply by 1ByteBit** [reply]
Will do!
